app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] Please specify the `foreign_key` for likes and comments `has_many` relationship just like you did for posts so that Rails knows where to look 👍